azure-pipelines.yml

```diff
@@ -2,90 +2,147 @@ trigger:
-        inputs:
-          SourceFolder: BlueTracker.SDK.Performance
-          Contents: |
-            *.nuspec
-            bin\**
-            obj\**
-          TargetFolder: '$(build.artifactstagingdirectory)'
-        condition: succeededOrFailed()
-
-      # Trigger when called from release pipeline, IS master
-      - task: PublishBuildArtifacts@1
-        displayName: 'Master Branch: Publish Artifact'
-        inputs:
-          PathtoPublish: '$(build.artifactstagingdirectory)'
-        condition: and(succeeded(), eq(variables['Build.sourceBranch'], 'refs/heads/master'))
-
-      # Trigger when called from build pipeline, NOT master
-      - task: PublishBuildArtifacts@1
-        displayName: 'Any Branch: Publish Artifact'
-        inputs:
-          PathtoPublish: '$(build.artifactstagingdirectory)'
-        condition: and(succeededOrFailed(), ne(variables['Build.sourceBranch'],'refs/heads/master'))
+stages:
+  - stage: Build
+    jobs:
+      - job: Build
+        displayName: 'Build'
+        pool: 'Fleet-Build'
+        strategy:
+          matrix:
+            Release-Any:
+              Platform: 'Any CPU'
+              BuildConfiguration: 'Release'
+        workspace:
+          clean: outputs
+
+        steps:
+          - checkout: self
+            clean: true
+
+          - task: GitVersion@5
+            displayName: 'GitVersion'
+            inputs:
+              updateAssemblyInfo: true
+
+          - task: NuGetToolInstaller@1
+            displayName: 'Use NuGet 5.6.0'
+            inputs:
+              versionSpec: 5.6.0
+
+          - task: NuGetCommand@2
+            displayName: 'NuGet restore'
+            inputs:
+              restoreSolution: BlueTracker.SDK.sln
+
+          - task: VSBuild@1
+            displayName: 'Build solution BlueTracker.SDK.sln'
+            inputs:
+              solution: BlueTracker.SDK.sln
+              vsVersion: '16.0'
+              platform: '$(Platform)'
+              configuration: '$(BuildConfiguration)'
+              clean: true
+
+          - task: MSBuild@1
+            displayName: 'NuGet package'
+            inputs:
+              solution: BlueTracker.SDK.sln
+              msbuildVersion: '16.0'
+              platform: '$(Platform)'
+              configuration: '$(BuildConfiguration)'
+              msbuildArguments: '/t:pack /p:PackageOutputPath="$(Build.ArtifactStagingDirectory)" /p:NoPackageAnalysis=true /p:PackageVersion=$(GitVersion.FullSemVer)'
+              maximumCpuCount: true
+
+          # Trigger when called from build pipeline, NOT master
+          - task: NuGetCommand@2
+            displayName: 'NuGet push'
+            inputs:
+              command: push
+              packagesToPush: '$(Build.ArtifactStagingDirectory)/*.nupkg'
+              publishVstsFeed: 'c69ffa3a-dc58-431c-afe0-12f465fcb265'
+            condition: and(succeeded(), ne(variables['Build.sourceBranch'], 'refs/heads/master'))
+
+          - task: CopyFiles@2
```

Comment:
The only artifact that is needed in the later steps is the nuget package. This and the following two tasks can be simplified.

Reply:
This `copy files` step is no longer needed as the `NuGet package` step already publishes the package to the `ArtifactsStagingDirectory`.

Reply:
Solved in: 5d52447